appscale/tools/node_layout.py

```diff
@@ -737,7 +740,20 @@ def from_locations_json_list(self, locations_nodes_list):
             # Locations JSON is incorrect if we get here.
             return None
           break
-
+    if len(nodes_copy) > 0:
+      for open_node in open_nodes:
+        for _, node in enumerate(nodes_copy):
```

Comment:
If you don't need the index, you probably don't need enumerate. Also, it looks like this is always just popping the first item from the list, so I'm not sure why iterating over `nodes_copy` is necessary.